techniques/css/C44.html

```diff
@@ -0,0 +1,94 @@
+		
+		<section id="examples">
+			<h2>Examples</h2>
+			<p>The following example can be seen as a <a href="../../working-examples/css-undersized-target-spacing/">working example</a>.</p>
+
+			<section class="example">
+				<h3>Adding spacing to undersized targets</h3>
+				<p>This example shows a situation where targets are smaller than 24 CSS pixels wide and 24 CSS pixels high. To meet Success Criterion 2.5.8, Target Size (Minimum), when a 24 CSS pixel diameter circle is centered on the bounding box of each target, the circles do not intersect another target or the circle for another undersized target. In this example, each target is 20 &#215; 20 CSS pixels, and a 4 CSS pixel gap is added between each target's parent container to create the minimum required spacing.</p>
+				<figure>
+					<img src="img/adding-spacing-undersized-targets.png" alt="a line of four interactive icons: back, forwards, email, home." height="21" width="102">
+					<figcaption>An example of inline undersized targets</figcaption>
+				</figure>
+				<h4>The <abbr title="HyperText Markup Language">HTML</abbr></h4>
+<pre><code class="language-html">&lt;nav aria-label="controls"&gt;
+  &lt;ol class="navigation-controls"&gt;
+    &lt;li&gt;&lt;a href="/previous-page/"&gt;&lt;img alt="Back" src="arrow-back.svg"&gt;&lt;/a&gt;&lt;/li&gt;
+    &lt;li&gt;&lt;a href="/next-page/"&gt;&lt;img alt="Forward" src="arrow-forward.svg"&gt;&lt;/a&gt;&lt;/li&gt;
+    &lt;li&gt;
+      &lt;a href="mailto:c42@example.com?subject=WCAG%20C42%20Technique"&gt;
+        &lt;img alt="Email" src="email.svg"&gt;
+      &lt;/a&gt;
+    &lt;/li&gt;
+    &lt;li&gt;&lt;a href="/"&gt;&lt;img alt="Home" src="home.svg"&gt;&lt;/a&gt;&lt;/li&gt;
+  &lt;/ol&gt;
+&lt;/nav&gt;</code></pre>
+			<h4>The CSS</h4>
+<pre><code class="language-css">.navigation-controls{
+  display: flex;
+  gap: 4px;
+  line-height: 0;
+  list-style: none;
+  padding: 0;
+}
+
+.navigation-controls a{
+  display: inline-block;
+  line-height: 0;
+}</code></pre>			
+
+			</section>
+		</section>
+		<section id="tests">
+			<h2>Tests</h2>
+			
+			<section class="test-procedure">
+				<h3>Procedure</h3>
+				<p>For each undersized target that has no equivalent, that isn't inline or otherwise constrained by the line-height of non-target text, that hasn't had its size modified by the author, and whose presentation isn't essential:</p>
+				<ol>
+					<li>Draw a 24px diameter circle centered on the target's bounding box.</li>
+					<li>Check that the circle from #1 does not intersect any other target.</li>
+					<li>Check that the circle from #1 does not intersect any other undersized target's circle.</li>					
+				</ol>
+			</section>
+			<section class="test-results">
+				<h3>Expected Results</h3>
+				<ul>
+					<li>#2 and #3 are true.</li>
+				</ul>
+			</section>
+		</section>
+		<section id="related">
+			<h2>Related Techniques</h2>
+			<ul>
+				<li><a href="C42.html">Using CSS to ensure sufficient target size</a></li>
```

Comment:
For consistency with other techniques:

```suggestion
				<li><a href="C42">C42</a></li>
```